lib/prawn/embedded_files/embedded_file.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'digest/md5'
+
+module Prawn
+  module EmbeddedFiles
+    class EmbeddedFile
+      attr_reader :chksum
+
+      def initialize(data, options = {})
+        @creation_date = options[:creation_date]
+        unless @creation_date.instance_of?(Time)
```

Comment:
Why not `kind_of?(Time)` here and on line 17?

Reply:
Sure, `kind_of?` seems more appropiate